modules/button/react/stories/stories_IconButton.tsx

```diff
@@ -224,6 +224,51 @@ storiesOf('Components|Buttons/Button/React/Icon Button', module)
       </div>
     );
   })
+  .add('Toolbar Square', () => {
+    const [toggled, setToggled] = React.useState<boolean | undefined>();
+    const handleToggle = () => {
+      setToggled(!toggled);
+    };
+
+    return (
+      <div className="story">
+        <h3>Small Toolbar Square</h3>
+        <IconButton
+          aria-label="Activity Stream"
+          icon={activityStreamIcon}
+          variant={IconButton.Variant.ToolbarSquare}
+        />
+        <IconButton
+          aria-label="Activity Stream"
+          icon={activityStreamIcon}
+          disabled={true}
+          variant={IconButton.Variant.ToolbarSquare}
+        />
+        <h3>Medium Toolbar Square</h3>
+        <IconButton
+          aria-label="Activity Stream"
+          icon={activityStreamIcon}
+          size={IconButton.Size.Medium}
```

Comment:
Why have a medium version if it's not supposed to be medium?

Reply:
Also your `getIconButtonSize` function doesn't seem to be working because this is still rendering a larger icon.